recipes/python-decouple/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "python-decouple" %}
+{% set version = "3.1" %}
+{% set sha256 = "1317df14b43efee4337a4aa02914bf004f010cd56d6c4bd894e6474ec8c4fe2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - mock
```

Comment:
It seems like `mock`, `pytest`, `tox`, `docutils` and `pygments` are all testing requirements and aren't actually needed to run this package. If this is the case, please remove these from the `run` section.

Reply:
@synapticarbors thank you for the advice! :+1: